tests/RequestHelpers.spec.ts

```diff
@@ -0,0 +1,95 @@
+import Ribbit from '../src';
+import {buildBody, buildUrl} from '../src/RequestHelpers';
+import {baseUrl} from './helpers';
+
+describe('That the buildBody() function works as expected', () => {
+    const api = new Ribbit();
```

Comment:
It says `api` is unused. I believe it is as well.

Reply:
Removed